tests/interfaces/test_torch_qnode.py

```diff
@@ -1644,7 +1730,7 @@ def circuit():
         assert result[2].dtype == torch.int64
 
 
-qubit_device_and_diff_method_and_mode = [
+qubit_device_and_diff_method_and_grad_on_execution = [
     ["default.qubit", "backprop", "forward"],
     ["default.qubit", "finite-diff", "backward"],
     ["default.qubit", "parameter-shift", "backward"],
```

Comment:
The `grad_on_execution` would make more sense here to be booleans rather than `"forward"` or `"backward"`

Reply:
Good catch I missed it!.